hcache/compr_lz4.c

```diff
@@ -0,0 +1,132 @@
+/**
+ * @file
+ * Header cache compression
+ *
+ * @authors
+ * Copyright (C) 2019 Tino Reichardt <milky-neomutt@mcmilk.de>
+ *
+ * @copyright
+ * This program is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation, either version 2 of the License, or (at your option) any later
+ * version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/**
+ * @page LZ4 compression
+ *
+ * Use LZ4 header cache compression for database backends.
+ */
+
+#include "config.h"
+
+#ifdef HAVE_LZ4_H
+#include <lz4.h>
+
+#include "muttlib.h"
+#include "compr.h"
+#include "hcache.h"
+#include "mutt/mutt.h"
+#include "mutt/memory.h"
```

Comment:
Please double check your includes:
1. what do you need muttlib.h for
2. mutt/mutt.h includes mutt/memory.h already